components/BankPayouts/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import Image from 'next/image'
+import HandWithPenny from 'public/images/handWithPenny.png'
+import LzButton from 'components/UI/button'
+import { FaAngleRight } from 'react-icons/fa'
+import { periphery } from './data'
+import LzInput from 'components/UI/input'
+
+interface MyProps {
+  title: string
+  desc: string
+  label: string
+  type: string
+  id: string
+}
+
+export const PayoutLHS = ({ title, desc, label, id, type }: MyProps) => {
+  return (
+    <>
+      <h1 className='heading-3-b lg:heading-2-b'>{title}</h1>
+      <p className='paragraph-2'>{desc}</p>
+      <form className='flex flex-col space-y-6'>
+        <LzInput type={type} label={label} id={id} />
+        <LzButton
+          variant='pri'
+          rightIcon={<FaAngleRight />}
```

Comment:
You don't need to pass a value to the `rightIcon`. There's already an icon in the component. pass the prop without a value. Please look at the button component to see how it works